build/azure-pipeline.stable.yml

```diff
@@ -0,0 +1,44 @@
+trigger:
+  branches:
+    include:
+      - main
```

Comment:
We'll need to discuss the release process in standup, since its manual we can ignore this for now.
This PR primarily focuses on the pre-release change and having the YML ready